demos/client-example-server/src/app.ts

```diff
@@ -55,8 +57,13 @@ async function main() {
     const uri = await memoryServerSetup()
     console.log('mongo uri', uri)
     const mongoose = connectionFactory(uri)
-    if (!process.env.PROSOPO_SITE_PRIVATE_KEY) {
-        throw new Error('No private key found')
+    if (!process.env.REACT_APP_SERVER_MNEMONIC) {
```

Comment:
What's the reason for the env variable name? All env vars should now start with PROSOPO

Reply:
I assume comparing and making changes vs an out of date file. Should be fixed now